src/index.ts

```diff
@@ -1,12 +1,14 @@
 /** @since 0.1.0 */
 
 import { pipe } from 'fp-ts/function';
-import * as M from 'fp-ts/Map';
 import * as A from 'fp-ts/Array';
+import * as E from 'fp-ts/Either';
 import * as O from 'fp-ts/Option';
-import { Option } from 'fp-ts/Option';
-import * as S from 'fp-ts/Set';
-import { Eq, struct } from 'fp-ts/Eq';
+import * as Cod from 'io-ts/Codec';
+import * as Dec from 'io-ts/Decoder';
```

Comment:
a detail: Personally I prefer to import the main type of a module unqualified. So we can write just `Decoder` instead of `Dec.Decoder`. Like for `Option`.